samples/RaspberryPiInstructions.md

```diff
@@ -1,70 +1,129 @@
+* [Start ssh automatically at boot time](https://raspberrypi.stackexchange.com/questions/1747/starting-ssh-automatically-at-boot-time).
 
-## Getting the app to run on the Pi.
+## Installing the .NET Core SDK on Rasberry Pi for Linux ARM32/ARM64
 
-### Linux
+The best way to install .NET Core on ARM platforms is currently via curl. We are working on other options.
 
-* Install [Linux](https://www.raspberrypi.org/downloads/) on your Pi.
+Note: The pattern documented below is the same one we use for [Docker](https://github.com/dotnet/dotnet-docker). You can look at .NET Core Dockerfiles to learn more about specific installation needs.
 
-* Install the [platform dependencies from your distro's package manager](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md) for .NET Core. .NET Core depends on some packages from the Linux package manager as prerequisites to running your application.
+1. Download .NET Core via curl (choose the version that works for you):
+  * 2.2 ARM32: `curl -SL --output dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.2.103/dotnet-sdk-2.2.103-linux-arm.tar.gz`
+  * 3.0 ARM32: `curl -SL --output dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotnet/Sdk/3.0.100-preview-010184/dotnet-sdk-3.0.100-preview-010184-linux-arm.tar.gz`
+  * 3.0 ARM64: `curl -SL --output dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotnet/Sdk/3.0.100-preview-010184/dotnet-sdk-3.0.100-preview-010184-linux-arm64.tar.gz`
 
-For Raspbian [Debian 9 Jessie](https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?tabs=netcore2x#install-net-core-for-debian-8-or-debian-9-64-bit) you need to do the following:
-```
-sudo apt-get update
-sudo apt-get install curl libunwind8 gettext apt-transport-https
+  Note: The latest download links are available at the [.NET Core Download page](https://dotnet.microsoft.com/download/archives).
+2. Unpack .NET Core to a global location:
+  * `sudo mkdir -p /usr/share/dotnet`
+  * `sudo tar -zxf dotnet.tar.gz -C /usr/share/dotnet`
+3. Symbolically link dotnet into a path location:
+  * `sudo ln -s /usr/share/dotnet/dotnet /usr/bin/dotnet`
+
+You should be able to run .NET Core at this point:
+
+```console
+dotnet
 ```
 
-* Copy your app, i.e. whole `publish` directory mentioned above, to the Raspberry Pi and execute run `./helloworld` to see `Hello World!` from .NET Core running on your Pi! (make sure you `chmod 755 ./helloworld`)
+## Installing the .NET Core SDK on Raspberry Pi on Win10 IoT Core
+
+Install [Windows 10 IoT Core](https://docs.microsoft.com/en-us/windows/iot-core/getstarted) on your Pi.
+
+Install the [.NET Core 3.0 SDK for Windows ARM32](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+
+## Creating and Running an Application on the Pi
+
+It is easy to run an application on the Pi. The following commands are for Linux, but switching the casing will enable the same commands to work on Windows.
```

Comment:
> switching the casing will enable the same commands to work on windows

I'm not sure I follow this statement?  What casing are you referring to?

Reply:
Crap, I meant the slash direction. I'll fix that.